.github/workflows/pr.yaml

```
@@ -73,7 +76,55 @@ jobs:
 
       - name: Test examples
         run: yarn test:examples
-      
+
+  build-lint-test-example-widgets-with-sdk:
+    name: Build, Lint & Test Example Widgets with latest SDK
+    runs-on: ubuntu-latest-8-cores
+    env:
+      NODE_OPTIONS: --max-old-space-size=14366
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+
+      - name: setup
+        uses: ./.github/actions/setup
+
+      - name: Setup playwright
+        uses: ./.github/actions/setup-playwright
+
+      - name: Get current SDK version string
+        uses: actions/github-script@v7.0.1
+        with: 
+          script: |
+            const { data: latestRelease } = await github.rest.repos.getLatestRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            });
+            console.log("Current SDK version: ", latestRelease.tag_name);
+            return latestRelease.tag_name;
+          result-encoding: string
+        id: version
+
+      - name: Install SDK at current version
+        run: yarn workspace @examples/sdk-load-widgets-with-nextjs add @imtbl/sdk@${{ steps.version.outputs.result }}
```

This could probably be cleaned up a bit.
The npm view command returns the latest npm version of a package, which should be matching the github version anyway 

```suggestion
      - name: Install SDK at current version
        run: yarn workspace @examples/sdk-load-widgets-with-nextjs add @imtbl/sdk@$(npm view @imtbl/sdk version)
```

---

Much cleaner, thanks.